next-cloudinary/src/components/CldUploadWidget/CldUploadWidget.types.ts

```diff
@@ -8,6 +8,54 @@ export interface CldUploadWidgetResults {
   info?: string | object;
 }
 
+export type CldUploadWidgetDetsroyInstanceMethodOptions = {
+  removeThumbnails: boolean;
+}
+
+export type CldUploadWidgetCloseInstanceMethodOptions = {
+  quiet: boolean;
+}
+
+export type CldUploadWidgetOpenInstanceMethodOptions = {
+  files: URL[];
```

Comment:
question about this one - i believe URL is likely the correct type here, but that means someone can't simply add a string?

i tried adding to test out and getting a type error. what do you think about allowing to be a URL or a string? i think a string would help trip up less people but if someone wants to use a URL instance (assuming thats what it is), its still accessible

<img width="704" alt="image" src="https://github.com/colbyfayock/next-cloudinary/assets/1045274/3dc94702-2123-4750-860a-711c604a114d">

Reply:
so actually, i think we'll NEED to do this and remove URL

the issue appears to be that because the way the widget works, it uses postMessage via communication between an iframe and it appears to not be serializable

<img width="570" alt="image" src="https://github.com/colbyfayock/next-cloudinary/assets/1045274/8f42a103-0348-48fe-9bed-3dfc5a7052de">

that's a limitation of the widget unfortunately

Reply:
Got it, in that case I've created a [custom URL type](https://github.com/msk4862/next-cloudinary/blob/b7739947bba2daa79b34afa78e308780fcc57943/next-cloudinary/src/components/CldUploadWidget/CldUploadWidget.types.ts#L6) that will do just basic checks for a URL.